### PR TITLE
Add how to build ironic images section

### PR DESCRIPTION
**Pull request by elfosardo**
Closes https://github.com/metal3-io/metal3-docs/issues/231

### CODE REVIEW

File: docs/user-guide/src/ironic/ironic-container-images.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```bash
+git clone https://github.com/metal3-io/ironic-image.git
+cd ironic-image
+docker build . -f Dockerfile
+```
+
+In some cases a **make** sub-command is provided to build the image using docker, usually **make docker**
+
+## Build ironic-image from source
```
Comment (Rozzii) [inline docs/user-guide/src/ironic/ironic-container-images.md:26]: Add a short description about the patching too please.

**Review by Rozzii — CHANGES_REQUESTED** [review]

**Review by Rozzii — APPROVED** [review]

**Comment by Rozzii** [comment]
Thank you for your contribution @elfosardo !
/lgtm

File: docs/user-guide/src/ironic/ironic-container-images.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Ironic Container Images
+
+The currently available ironic container images are listed below.
+
+| Name and link to repository | Content/Purpose |
+| --- | --- |
+| [ironic-image](https://github.com/metal3-io/ironic-image) | Ironic api and conductor / Ironic Inspector / Sushy tools |
```
Comment (furkatgofurov7) [inline docs/user-guide/src/ironic/ironic-container-images.md:7]: Is not is also:
```suggestion
| [ironic-image](https://github.com/metal3-io/ironic-image) | Ironic api and conductor / Ironic Inspector / Sushy tools and virtualbmc |
```
?

**Review by furkatgofurov7 — COMMENTED** [review]
Great work, found one small nit:

File: docs/user-guide/src/ironic/ironic-container-images.md (hunk 2 above)
Comment (elfosardo) [reply docs/user-guide/src/ironic/ironic-container-images.md:7]: you are totally right, I will update it

**Comment by furkatgofurov7** [comment]
Can Ironic repo maintainers take a look at this and help to merge it? 
/cc @dtantsur @bfournie @iurygregory

**Review by smoshiur1237 — APPROVED** [review]
LGTM

**Review by Rozzii — APPROVED** [review]

**Comment by furkatgofurov7** [comment]
@elfosardo can you please fix the rebase issues, we had some PRs landing recently, thanks!

**Review by furkatgofurov7 — APPROVED** [review]
/approve

/cc @Rozzii

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Pull request merged**